lib/index.js

```diff
@@ -0,0 +1,169 @@
+require("dotenv").config();
+const { OpenAI } = require("openai");
+const fs = require("fs");
```

Comment:
We import  a bunch of packages including

- fs - for handling file system. We need it for reading and writing files created by our code
- axios - used to make network requests
- cheerio - used to parse HTML pages
- pretty - used to format the output of cheerio into a nice readable string